python/chronos/src/bigdl/chronos/forecaster/tf/lstm_forecaster.py

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from bigdl.chronos.forecaster.tf.base_forecaster import BaseTF2Forecaster
+from bigdl.chronos.model.VanillaLSTM_keras import model_creator
+
+
+class LSTMForecaster(BaseTF2Forecaster):
```

Comment:
add a example usage as in pytorch LSTMForecaster
https://github.com/intel-analytics/BigDL/blob/315dd847dfde9a52d4733cc51380104a81e43250/python/chronos/src/bigdl/chronos/forecaster/lstm_forecaster.py#L23-L36